app/assets/stylesheets/layout.scss

```diff
@@ -1270,16 +1266,6 @@ form {
   vertical-align: top;
 }
 
-.user-permissions {
```

Comment:
Not a big deal, but what do you think about removing the class `user-permissions` from the following views?
* app/views/account/show.html.erb
* app/views/management/_user_permissions.html.erb
* app/views/verification/letter/new.html.erb
* app/views/verification/residence/new.html.erb

It seems now that class is useless 🤔

Reply:
Interesting! I'd vote for keeping the class, at least for now. There might be people using custom styles for this class, and removing it would make the element harder to style :thinking:.

Reply:
Ok, nice catch!